ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -53,6 +53,47 @@ tests:
   container:
     clone: true
     from: golangci-lint
+- as: check-commits-count
+  commands: ci/prow/check-commits-count
+  container:
+    clone: true
+    from: src
+  optional: true
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.11"
+  steps:
+    test:
+    - as: e2e-incluster-build
+      cli: latest
+      commands: GOFLAGS= ci/prow/e2e-incluster-build
+      from: src
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    - as: e2e-prebuilt-kernel-module
+      cli: latest
+      commands: GOFLAGS= ci/prow/e2e-prebuilt-kernel-module
+      from: src
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    - as: e2e-crd-variable
```

Comment:
Same here. I think we should set a different test for it (and not another step in the same test).